src/ESPiLight.cpp

```diff
@@ -46,19 +46,21 @@ volatile unsigned long ESPiLight::_lastChange =
 volatile uint8_t ESPiLight::_nrpulses = 0;
 int16_t ESPiLight::_interrupt = NOT_AN_INTERRUPT;
 
-uint8_t ESPiLight::minrawlen = 5;
-uint8_t ESPiLight::maxrawlen = MAXPULSESTREAMLENGTH;
-uint16_t ESPiLight::mingaplen = 5100;
-uint16_t ESPiLight::maxgaplen = 10000;
+uint8_t ESPiLight::minrawlen = std::numeric_limits<uint8_t>::max();
+uint8_t ESPiLight::maxrawlen = std::numeric_limits<uint8_t>::min();
+uint16_t ESPiLight::mingaplen = std::numeric_limits<uint16_t>::max();
+uint16_t ESPiLight::maxgaplen = std::numeric_limits<uint16_t>::min();
+uint16_t ESPiLight::minpulselen = 80;
+uint16_t ESPiLight::maxpulselen = 16000;
```

Comment:
It does not yet work as expected. The initialization needs to be done in `calc_lengths()`. Otherwise they will not be updated with `ESPiLight::limitProtocols()`.

Reply:
It works fine for me and the values are updated:
```
minrawlen: 26
maxrawlen: 148
mingaplen: 3600
maxgaplen: 35020
minpulselen: 80
maxpulselen: 35020
```

Reply:
Did you tried the `test_proto_limit` example? The values are only get updated if there is a more strict requirement (smaller for `min*`, larger for `max*`) but never in the other direction.

`mingaplen` is an exception, because it will be initialized in `calc_lengths()`.

Reply:
Ohh I did not know that this was required, then we need to reset the values before updating them

Reply:
Yes, otherwise, `calc_lengths()` in `ESPiLight::limitProtocols()` makes almost no sense.